Tests/Cosmos.Compiler.Tests.Bcl/System/MathTest.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Cosmos.Compiler.Tests.Bcl.System
 {
-    class MathTest
+    internal class MathTest
     {
```

Comment:
"internal" qualifier not needed

Reply:
Even though `internal` is the default, I think that access modifiers should be explicitly specified.